### ISSUE TITLE
Django-CMS 3.2.2 Menu Locks & Freezes

### ISSUE THREAD

**Issue by bartmika** [divio/django-cms#5076]
I can't reproduce this issue on **Django-CMS 3.2.1** but now that I'm running version **3.2.2** I noticed this weird error. When I add a new page or application from the Pages section then the menu disappears and everything on the page is locked up (nothing is clickable).

**Comment by mkoistinen** [comment]
@bartmika Did you get any stacktraces? Can you perhaps make a short screencast of what's going on? I can't reproduce this locally.

**Comment by jsma** [comment]
I'm seeing this too. Just upgraded to 3.2.2 and the latest text plugins and now the plugin dropdown doesn't work:
<img width="266" alt="screen shot 2016-03-09 at 1 58 25 pm" src="https://cloud.githubusercontent.com/assets/24574/13652060/06293588-e5ff-11e5-96ce-e427809979e4.png">

I have no familiarity with the inner workings of CKE or how the CMS Plugin menu gets populated, but if you give me some troubleshooting pointers, I'll try to track this down.

**Comment by czpython** [comment]
@jsma Hi there, are you using the latest djangocms-text-ckeditor? (if not, which version?)

**Comment by bartmika** [comment]
@mkoistinen My apologies I do get this error in the browser console.

![screen shot 2016-03-09 at 5 43 54 pm](https://cloud.githubusercontent.com/assets/5767602/13653457/ec49738c-e61f-11e5-96e0-8c14a8afdb59.png)

For stack trace, I'll get you it when I reproduce it again. This issue appears intermittent.

**Comment by jsma** [comment]
@czpython yes, using 2.9.0. (It now appears I'm talking about a different issue than @bartmika, sorry for hijacking) 

It appears to be some issue with CMS_PLACEHOLDER_CONF. If I comment out my settings, the CMS Plugins menu reappears. For example, I have this entry:

    'homepage-intro': {
            'plugins': ['TextPlugin'],
            'limits': {
                'TextPlugin': 1
            }
        },

If I comment out the 'plugins' item, it works. If I add this, it works:

    'text_only_plugins': [
            'LinkPlugin',
            'FilerImagePlugin',
            'FilerFilePlugin',
        ]

Am I crazy? I thought the `text_only_plugins` was optional and that my original configuration without it was working at some point.

**Comment by bartmika** [comment]
Hi @mkoistinen, I've reproduced this error. I've basically created a new page, saved the contents with english and then created another language version using arabic 'ar'. I then go to that page after saving it and in the console log i I get:

  ```python
  /Users/blahblah/Developer/lalalala/albayanf/py-albayanf/env/lib/python3.4/site-packages/django/template/base.py:827: RemovedInDjango19Warning: `request.REQUEST` is deprecated, use `request.GET` or `request.POST` instead.
  current = getattr(current, bit)

/Users/blahblah/Developer/lalalala/albayanf/py-albayanf/env/lib/python3.4/site-packages/django/core/handlers/wsgi.py:126: RemovedInDjango19Warning: `MergeDict` is deprecated, use `dict.update()` instead.
  self._request = datastructures.MergeDict(self.POST, self.GET)
  ```

**Comment by czpython** [comment]
Hello @bartmika,
Are you able to reproduce this with the latest version (3.4.2)?

@jsma 
The issue you're referring to is a known issue for django CMS text ckeditor when using the `HTMLField`.

**Comment by bartmika** [comment]
My appoligies @czpython, I won't be able to confirm this intermittent issue at the present moment. Perhaps we can close this issue now and in the future if I confirm this issue reappears under 3.4.2, I'll reopen this issue. Sounds good?
